docs/generated/packages/detox/executors/test.json

```diff
@@ -24,18 +24,21 @@
           "ios.sim.release",
           "android.emu.debug",
           "android.emu.release"
-        ]
+        ],
+        "x-priority": "important"
```

Comment:
If a flag is already required, we don't need to mark it as `important` too. it's not that bad if you do but just inconsistent and duplicate.

Reply:
removed